.github/workflows/testpythonpackage.yml

```diff
@@ -0,0 +1,55 @@
+# SPDX-FileCopyrightText: 2021-2024 The Ikarus Developers mueller@ibb.uni-stuttgart.de
+# SPDX-License-Identifier: CC0-1.0
+
+name: Test Python Package
+on:
+  push:
+    paths-ignore:
+      - 'docs/**'
+      - '.github/workflows/ghpages.yml'
+      - '.github/workflows/createDockerContainer.yml'
+      - '**.md'
+
+  pull_request:
+    types: [opened]
+    branches:
+      - main
+    paths-ignore:
+      - 'docs/**'
+      - '.github/workflows/ghpages.yml'
+      - '.github/workflows/createDockerContainer.yml'
+      - '**.md'
+
+jobs:
+  build-package:
+    runs-on: ubuntu-latest
+    container:
+      image: ikarusproject/ikarus-dev:latest
+      options: --memory-swap="20g" --memory="20g" --cpus="2" --user root
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install twine and scikit-build
+        run: |
+          /dune/dune-common/build-cmake/run-in-dune-env pip install twine scikit-build
```

Comment:
Maybe you can check if the venv is active already or activate it. Then you dont have to prepend everythign with `/dune/dune-common/build-cmake/run-in-dune-env`.

Reply:
Good idea, I think this might be the reason why the tests are quite slow. 
I think I will also add a file called `testall.py` and let this run

Reply:
Instead of testall.py you can also write this as a short inline bash script then we dont have to add a new file

Reply:
```bash
for i in $( find . -maxdepth 1 -name '*.py' );  do./"$i"; done;
```
something like that